Add seedrandom to support seeded random number generation for the visual tests

jleft opened the pull request:
Many visual tests rely upon `fc.dataGenerator`, which uses `Math.random()` to generate the data values. This is great for producing varied data, but sometimes, when debugging for example, it it is useful to use the same dataset.

This pull request adds [seedrandom](https://github.com/davidbau/seedrandom) as a development dependency, which allows `Math.random()` to be seeded and `fc.dataGenerator` to produce consistent values. This can be included in the visual tests by running `grunt build:visual-tests-debug`.

Does this seem like a good idea? Is this a reasonable approach?

ColinEberhardt commented:
Definitely a worthwhile addition. Although I wonder whether it is worth adding another test target just for this? I'd be happy for the visual tests to always run with the seeded random number.

@chrisprice any thoughts?

chrisprice commented:
Maybe we could always include seedrandom in the page but seed it with the query string from the URL if it exists?

jleft commented:
Thanks for the feedback :)

> Although I wonder whether it is worth adding another test target just for this?

Agreed, I didn't really like adding the extra target - I've removed this.

> Maybe we could always include seedrandom in the page but seed it with the query string from the URL if it exists?

Nice idea, I've updated to include this.

chrisprice commented:
LGTM, thanks!
